#EmgArray message for the Thalmic Myo, which has 8 EMG sensors 
#arranged about the arm
int8 data1
int8 data2
int8 data3
int8 data4
int8 data5
int8 data6
int8 data7
int8 data8
